src/effects/builtin/common/generatoreffect.h

```diff
@@ -3,17 +3,27 @@
  */
 #pragma once
 
+#include "context/iglobalcontext.h"
+#include "playback/iplayback.h"
+#include "modularity/ioc.h"
```

Comment:
please move it to the top

Reply:
Why ?

Reply:
But certainly, a header file should not rely on other header files being included before itself, right ?
And order of importance: I don't know what that is and I don't want to spend time thinking about it when I include.

Reply:
it's more about code cleanliness